examples/deployment-oss-min/nginx-ingress-controller.yaml

```diff
@@ -7,9 +7,9 @@ spec:
   type: deployment
   nginxPlus: false
   image:
-    repository: registry.hub.docker.com/nginx/nginx-ingress
-    tag: edge-ubi
+    repository: nginx/nginx-ingress
+    tag: 1.12.0-ubi
```

Comment:
Minor one: should we do the same in `deployment-plus-min` as well? considering we now specify the latest release tag in KIC master and release but in operator it remains the same i.e. `edge` in both master and release.